### PR TITLE
Actionable proposals by default

### PR DESCRIPTION
**Pull request by mstrasinskis**
# Motivation

In its current version, “actionable proposal” tab is on the right, and isn’t selected by default. This makes it feel like a secondary feature, and it is hidden by default, however most users only need to look at actionable proposals. Having the “history” is just a nice bonus in case someone is interested.

Since `All Proposals` is no longer a default option, multiple tests needed to be updated to select `All Proposals` before running.

**Update**: since displaying actionables doesn't add anything for users who aren't logged in, we've decided to show all proposals only for logged-out users (w/o the switcher). This adjustment streamlines the interface and ensures that users see relevant content based on their authentication status.
Because the actionable state isn't available in signedOut state, the `ActionableProposalsSignIn` isn't used anymore. It will be removed in a separate pr.

# Changes

- Make `actionable` a default segment store value.
- Change the order of the segments to make `Actionable` the first element.
- Add `actionableProposalsActiveStore` store (combines signIn status with segment selection).
- Reload actionable proposals after creating "dummy" proposals to improve the testing experience and ensure e2e tests pass.

# Tests

- `actionableProposalsActiveStore`
- Adjust current tests (they already cover the default selection).
- Not part of the feature:
  - Some outdated tests have been removed from `.../routes/Proposals.spec.ts`. Changing the initial tab selection caused them to fail, and upon fixing, it was discovered that they are obsolete since the `select-project-dropdown` has not been part of the Proposals component for some time. The removed test coverage will be added in a separate PR.
  - Removed navigation to proposals page duplication in `e2e/proposals.spec.ts`.

# Todos

- [x] Add entry to changelog (if necessary).

### CODE REVIEW

File: frontend/src/tests/e2e/proposals.spec.ts [hunk 1]
```diff
@@ -33,13 +33,16 @@ test("Test proposals", async ({ page, context }) => {
...
+    .getProposalsPo()
+    .getNnsProposalFiltersPo()
+    .getActionableProposalsSegmentPo()
+    .clickAllProposals();
+
   /*
    * Test proposal filters
    */
-  step("Open proposals list");
-  await appPo.goToProposals();
```
Comment (dskloetd) [inline frontend/src/tests/e2e/proposals.spec.ts:40]: Was this already not necessary? Or why is this removed now?

File: frontend/src/tests/lib/routes/Proposals.spec.ts [hunk 2]
```diff
@@ -48,67 +34,6 @@ describe("Proposals", () => {
 
   it("should render NnsProposals by default", () => {
     const { queryByTestId } = render(Proposals);
-    expect(queryByTestId("proposals-filters")).toBeInTheDocument();
-  });
-
-  describe("sns", () => {
```
Comment (dskloetd) [inline frontend/src/tests/lib/routes/Proposals.spec.ts:54]: What happened to this whole block?

File: frontend/src/tests/lib/stores/actionable-proposals-segment.store.spec.ts [hunk 3]
```diff
@@ -7,7 +7,7 @@ describe("actionable proposals segment store", () => {
   });
 
   it('should have "all" by default ', () => {
```
Comment (dskloetd) [inline frontend/src/tests/lib/stores/actionable-proposals-segment.store.spec.ts:9]: fix description

File: frontend/src/lib/stores/actionable-proposals-segment.store.ts [hunk 4]
```diff
@@ -14,12 +14,12 @@ export interface ActionableProposalsSegmentStore
 
 /**
  * A store that contains selected state of actionable proposals segment.
- * By default, it's "all" and not stored in LocalStorage.
+ * By default, it's "actionable" and not stored in LocalStorage.
```
Comment (dskloetd) [inline frontend/src/lib/stores/actionable-proposals-segment.store.ts:17]: Is it also default `"actionable"` when the user is not logged in?
That seems like it would be a worse experience than necessary.

Comment (mstrasinskis) [reply frontend/src/lib/stores/actionable-proposals-segment.store.ts:17]: This is the first version. We decided to improve it since there are some feedback.

File: frontend/src/tests/e2e/proposals.spec.ts (hunk 1 above)
Comment (mstrasinskis) [reply frontend/src/tests/e2e/proposals.spec.ts:40]: I just noticed that this was just done a couple of lines before. Do you think it make sense to leave it?

File: frontend/src/tests/lib/routes/Proposals.spec.ts (hunk 2 above)
Comment (mstrasinskis) [reply frontend/src/tests/lib/routes/Proposals.spec.ts:54]: These tests are obsolete, and they didn't fail just by pure chance. There is no project selector in the `Proposals` component anymore. I'll already started adding them back in a separate branch.

File: frontend/src/tests/lib/stores/actionable-proposals-segment.store.spec.ts (hunk 3 above)
Comment (mstrasinskis) [reply frontend/src/tests/lib/stores/actionable-proposals-segment.store.spec.ts]: Good catch, updated.

File: frontend/src/lib/stores/actionable-proposals-segment.store.ts (hunk 4 above)
Comment (dskloetd) [reply frontend/src/lib/stores/actionable-proposals-segment.store.ts:17]: Are you reconsidering this with Andrew?

File: frontend/src/tests/e2e/proposals.spec.ts (hunk 1 above)
Comment (dskloetd) [reply frontend/src/tests/e2e/proposals.spec.ts:40]: If it's not useful, I don't want to keep it. I just wanted to make sure I'm not missing something.

File: frontend/src/lib/stores/actionable-proposals-segment.store.ts (hunk 4 above)
Comment (mstrasinskis) [reply frontend/src/lib/stores/actionable-proposals-segment.store.ts:17]: Yes, thanks, the descriptions was updated.
